src/DummyData/DataTypes.ts

```diff
@@ -0,0 +1,27 @@
+export type FoodItem = {
```

Comment:
constant variables should be lowercase

Reply:
This is a type, not a constant variable. Shouldn't this be in PascalCasing?

Reply:
yes, ignore my comment then